tests/Jobs/Imports/ParseRockTheVoteReportTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use App\Jobs\Imports\ImportRockTheVoteRecord;
+use App\Jobs\Imports\ParseRockTheVoteReport;
+use Illuminate\Support\Facades\Bus;
+use Illuminate\Support\Facades\Storage;
+
+class ParseRockTheVoteReportTest extends TestCase
+{
+    /**
+     * Test that this job can parse a Rock The Vote CSV report.
+     *
+     * @return void
+     */
+    public function testParsesReport()
+    {
+        Bus::fake();
+        Storage::fake();
+
+        // Create a faked record that we can safely read & delete:
```

Comment:
Not sure if typo but sounds funny!
```suggestion
        // Create a fake record that we can safely read & delete:
```